util/memory/tracker.go

```diff
@@ -275,3 +275,26 @@ func (t *Tracker) BytesToString(numBytes int64) string {
 
 	return fmt.Sprintf("%v Bytes", numBytes)
 }
+
+// AttachToGlobalTracker attach the tracker to the global tracker
+// AttachToGlobalTracker should be called at the initialization for the session executor's tracker
+// If the tracker already have parent, we would directly return.
+func (t *Tracker) AttachToGlobalTracker(globalTracker *Tracker) {
```

Comment:
Some comments are nice to be added to `Tracker.children`, for explaining this field is not maintained if it acts as a global tracker for less mutex contention.